CondTools/SiPixel/plugins/FastSiPixelFEDChannelContainerFromQuality.cc

```diff
@@ -304,7 +306,7 @@ FastSiPixelFEDChannelContainerFromQuality::createFromSiPixelQuality(const SiPixe
     std::vector<PixelFEDChannel> disabledChannelsDetSet;
     std::vector<sipixelobjects::CablingPathToDetUnit> path = theFedCabling.pathToDetUnit(mod.DetID);
     unsigned int nrocs_inLink(0);
-    if (path.size() != 0) {
+    if (!path.empty()) {
```

Comment:
This check is useless, if independently of it you later divide by `nrocs_inLink` anyhow.
You can remove it, or (maybe better) throw some exception inside